a11y prs

bollwyvl opened the pull request:
## what kind of request is this?

- [x] a preview site on ReadTheDocs
  - [x] it only adds a `jupyak_config.toml`
- [ ] a `jupyak` code change
  - [ ] it adds new features, described below

## for preview sites

### how should a visitor use the preview site?

<!-- please describe the steps one should follow to evaluate -->

1. open a new **private** browser window
2. go to https://jupyak--16.org.readthedocs.build/en/16/_static/work/lite/lab/
3. see the _Launcher_
3. use <kbd>tab</kbb> one or more times to focus the _Python (pyodide)_ launcher item
4. press <kbd>enter</kbd> to start a new notebook from the launcher
5. press <kbd>esc</kbd> to enter _Command Mode_
6. use <kbd>shift+tab</kbb> a few times navigate back to _Close_ button for the activity
7. use <kbd>delete</kbd> to close the notebook
8. accept the dialog default with <kbd>enter</kbd>

### where is the main discussion?

https://github.com/t03857785/jupyterlab/issues/12

bollwyvl committed:
a11y prs

bollwyvl committed:
use a dangling lumino pr

bollwyvl committed:
try theirs

tonyfast commented on jupyak_config.toml:
```diff
@@ -0,0 +1,18 @@
+[repos]
+lumino.github.merge_with = [
+    "pull/655",
+]
```
```suggestion
    "pull/627",
]
```

tonyfast commented on jupyak_config.toml:
```diff
@@ -0,0 +1,18 @@
+    "pull/14766",
+    "pull/14854",
+    "pull/15222",
+    "pull/15347"
```
```suggestion
    "pull/15347",
    "pull/15048",
```

tonyfast reviewed (commented):
adds aria live pull requests to the build

tonyfast committed:
Update jupyak_config.toml

tonyfast committed:
Update jupyak_config.toml

tonyfast commented on jupyak_config.toml:
```diff
@@ -0,0 +1,20 @@
+[repos]
+lumino.github.merge_with = [
+    "pull/655",
+    "pull/627",
```
```suggestion
```

tonyfast committed:
Update jupyak_config.toml

tonyfast committed:
kick a build

space is UI

bollwyvl committed:
Merge remote-tracking branch 'UPSTREAM/main' into a11y-prs

bollwyvl committed:
Merge remote-tracking branch 'origin/a11y-prs' into a11y-prs

bollwyvl committed:
Merge remote-tracking branch 'UPSTREAM/main' into a11y-prs

bollwyvl committed:
Merge remote-tracking branch 'UPSTREAM/main' into a11y-prs

bollwyvl committed:
Merge remote-tracking branch 'UPSTREAM/main' into a11y-prs

bollwyvl committed:
Merge remote-tracking branch 'UPSTREAM/main' into a11y-prs

bollwyvl committed:
yank gist

bollwyvl committed:
Merge remote-tracking branch 'UPSTREAM/main' into a11y-prs

bollwyvl committed:
Merge remote-tracking branch 'origin/a11y-prs' into a11y-prs

tonyfast committed:
Update jupyak_config.toml

update configuration based on  https://github.com/t03857785/jupyterlab/issues/12
